<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicUserManager</class>
 <widget class="QWidget" name="MusicUserManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicUserManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MusicUserManager{background:white;border:1px solid gray}</string>
  </property>
  <property name="currentIndex" stdset="0">
   <number>0</number>
  </property>
  <widget class="QLabel" name="userIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:b</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>15</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="logoffButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>83</y>
     <width>71</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>在线总时长：</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalTimeLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>108</y>
     <width>71</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>在线总等级：</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalLevelLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>105</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="modifiedName">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>45</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
